HEC.FDA.ModelTest/integrationtests/DefaultDataComputeOutcomes.cs

```diff
@@ -268,7 +268,7 @@ public void AnalyticalWithRegUnreg_ScenarioResults(double expectedResidentialMea
             //Act
             double actualResidentialMeanEAD = scenarioResults.MeanExpectedAnnualConsequences(impactAreaID1, residentialDamageCategory);
             double actualCommercialMeanEAD = scenarioResults.MeanExpectedAnnualConsequences(impactAreaID1, commercialDamageCategory);
-            double tolerance = 0.19;
+            double tolerance = 0.2;
```

Comment:
Should we reconsider whether this test is helpful?

Reply:
Likely. Where did the expected values come from?